Kickstarter-iOS/Views/RewardCardView.swift

```diff
@@ -277,11 +275,14 @@ public final class RewardCardView: UIView {
     self.setNeedsLayout()
   }
 
-  fileprivate func load(items: [String]) {
+  fileprivate func load(items: ([String], UIColor)) {
```

Comment:
Could we name the tuple values here like `fileprivate func load(items: (includedItems: [String], separatorBackgroundColo: UIColor)) {`, and then below refer to like `items.includedItems` and `items.separatorBackgroundColor`?